### ISSUE TITLE
如何控制稀疏率为25%

### ISSUE THREAD

**Issue by dewishyi** [xian-sh/MRNet#4]
Dear Author,

Hello! I truly admire the outstanding work you've done here. It's truly impressive. While I was diving into your code, I couldn't seem to find where the sparsity rate is set. The paper mentions that the optimal sparsity rate is 25%, but I'm not sure where this is implemented in the code. Could you kindly point me in the right direction? I'd really appreciate your help.

Best regards!

**Comment by xian-sh** [comment]
> Dear Author,
> 
> Hello! I truly admire the outstanding work you've done here. It's truly impressive. While I was diving into your code, I couldn't seem to find where the sparsity rate is set. The paper mentions that the optimal sparsity rate is 25%, but I'm not sure where this is implemented in the code. Could you kindly point me in the right direction? I'd really appreciate your help.
> 
> Best regards!

Thank you for your attention, the relevant code has been screenshot below, you see if it can solve your confusion, thank you!
![Image](https://github.com/user-attachments/assets/097e2790-4537-4ec3-a104-6ea8d6bc387d)

![Image](https://github.com/user-attachments/assets/684d2714-2d58-4c56-8421-2980c85ee6fe)

**Comment by dewishyi** [comment]
Thank you for your explanation. I still have two questions that I hope you can help me with:

1. Regarding the calculation of `overall_min = cfg.SOLVER.LOSS_RATIO * cfg.SOLVER.BATCH_SIZE * cfg.SOLVER.SPARSE_WEIGHT`, why is it computed in this way?

2. In the following code:
   ```
   loss += loss_sent + loss_vid + loss_sp
   if loss_sp >= overall_min:
       loss += loss_sp
   ```
   The `loss` has already included `loss_sp`, but then it checks the threshold and adds it again. Why is this done?

Your answers would be of great help to me.
Best regards!
